packages/xstate-graph/test/types.test.ts

```diff
@@ -52,28 +50,20 @@ describe('types', () => {
     >({});
 
     getMachineShortestPaths(machine, {
-      eventCases: {
-        FOO: [
-          {
-            value: 100
-          }
-        ]
-      }
+      events: [{ type: 'FOO', value: 100 }]
     });
   });
 
   it('`eventCases` should not allow unknown events', () => {
     const machine = createMachine<unknown, { type: 'FOO'; value: number }>({});
 
     getMachineShortestPaths(machine, {
-      eventCases: {
-        // @ts-expect-error
-        UNKNOWN: [
-          {
-            value: 100
-          }
-        ]
-      }
+      events: [
+        {
+          type: 'UNKNOWN',
+          value: 100
+        } as any
+      ]
```

Comment:
This is a type test - so it's whole point was to check for the error ;p
```suggestion
      events: [
        {
          // @ts-expect-error
          type: 'UNKNOWN',
          value: 100
        }
      ]
```